django/contrib/postgres/fields/array.py

```diff
@@ -88,6 +88,16 @@ def to_python(self, value):
             value = [self.base_field.to_python(val) for val in vals]
         return value
 
+    def from_db_value(self, value, expression, connection, context):
+        if value is None:
+            return value
+        return [
+            self.base_field.from_db_value(val, expression, connection, context)
+            if hasattr(self.base_field, 'from_db_value') else
```

Comment:
Wouldn't be nicer if `from_db_value` was declared on Field? Then you can call `from_db_value` without checking `hasattr`